heartofthealien/heartofthealien-1.2.2.recipe

```diff
@@ -0,0 +1,50 @@
+SUMMARY="Heart of Alien Redux"
+DESCRIPTION="Heart of the Alien 1.2.2 game port for BeOS"
+HOMEPAGE="http://hota.sourceforge.net/"
+COPYRIGHT="2004 Gil Megidish"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="8f1f525fcf50c4610302e870c85625c8ea2f33eb"
+SOURCE_URI="https://github.com/ytmytm/beos-heartofthealien/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="heartofthealien-$portVersion~$srcGitRev.tar.gz"
+SOURCE_DIR="BeOS-heartofthealien-$srcGitRev"
```

Comment:
Should be
```
SOURCE_DIR="beos-heartofthealien-$srcGitRev"
```
as I wrote in my first review. I did not write "BeOS". I wrote "beos"
(This is because the filesystem is case-sensitive.)